nautobot_chatops/dispatchers/slack.py

```diff
@@ -489,3 +491,22 @@ def select_element(self, action_id, choices, default=(None, None), confirm=False
     def text_element(self, text):
         """Construct a basic plaintext element."""
         return {"type": "plain_text", "text": text}
+
+
+class SlackSocketDispatcher(SlackDispatcher):
+    """Overrides to support Slack Socket Mode."""
+
+    def static_url(self, path):
+        """Return a url to access the static file."""
+        static_path = str(static(path))
+        if static_path.startswith("http"):
+            # Static is being provided by Django Storages
+            return static_path
+        if SLACK_SOCKET_STATIC_HOST:
+            # Use the settings provided Static Host.
+            return f"{SLACK_SOCKET_STATIC_HOST}{static_path}"
+        return None
+
+    def image_element(self, url, alt_text=""):
+        """Construct an image element that can be embedded in a block if URL is provided."""
+        return {"type": "image", "image_url": url, "alt_text": alt_text} if url else {}
```

Comment:
This seems like a change that would be safe enough to make to the base `SlackDispatcher`?

Reply:
Good point. I originally created this as I was just disabling the static_url for testing.